test/1_test_fullpath.js

```diff
@@ -1429,15 +1421,13 @@ contract('Voucher tests - UNHAPPY PATH', async (addresses) => {
       });
 
       //Check VoucherKernel state
-      const voucherStatusBefore = await contractVoucherKernel.vouchersStatus(
+      const voucherStatusBefore = await contractVoucherKernel.getVoucherStatus(
         tokenVoucherKey1
       );
       const transaction = await web3.eth.getTransaction(complainTx.tx);
       const transactionBlock = await web3.eth.getBlock(transaction.blockNumber);
       assert.isTrue(
-        voucherStatusBefore.cancelFaultPeriodStart.eq(
-          new BN(transactionBlock.timestamp)
-        )
+        voucherStatusBefore[4].eq(new BN(transactionBlock.timestamp))
```

Comment:
Please use constants instead of hard-coded indexes for voucherStatusBefore (same as remark above)

Reply:
Remaining hard-coded indexes at line 1501